### PR TITLE
Add support for multi-arc image mirrorig

### PR DESCRIPTION
**Pull request by StrongMonkey**

### CODE REVIEW

**Comment by StrongMonkey** [comment]
Address https://github.com/rancher/rancher/issues/21221.

**Comment by StrongMonkey** [comment]
@cloudnautique @paraglade changes to pushed to address a bug with tagging multiple arch.

File: scripts/docker-pull-pusher.sh [hunk 1]
```diff
@@ -1,21 +1,30 @@
 #!/bin/sh
 set -e
+set -x
```
Comment (cloudnautique) [inline scripts/docker-pull-pusher.sh]: I don't think this should be on by default.

**Comment by StrongMonkey** [comment]
Per @ibuildthecloud 's suggestion. Going to take a different approach.

**Pull request closed without merging**